src/provisioning-ofono.c

```diff
@@ -1230,10 +1278,9 @@ int provisioning_init_ofono(void)
  * Currently this assumes that only one modem in ofono if need to support more
  * it could be something like below
  */
-#ifdef MULTIMODEM
 	modem_list = g_hash_table_new_full(g_str_hash, g_str_equal,
-						NULL, remove_modem);
-#endif
+						NULL, remove_modems);
+
```

Comment:
I have made changes so that multiple times remove_modems is not called. Unable to push due to merge conflict will upload once it is resolved